<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addrChangeDlg</class>
 <widget class="QDialog" name="addrChangeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>77</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>源地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>77</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>地址信息：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>76</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>替换地址：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>替换范围</string>
   </property>
   <widget class="QRadioButton" name="radioButton_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>仅替换本地址</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>本控件地址</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>本画面所有地址</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>本工程所有地址</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_info">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_addrOld">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_addrNew">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sure">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addr">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>注意：该地址只能为内部地址</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../resourse.qrc">
     <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
